code/modules/antagonists/starfurybc/starfurybc.dm

```diff
@@ -0,0 +1,82 @@
+/datum/antagonist/starfuryop
+	name = "Syndicate Operative"
+	roundend_category = "starfury battlecruiser operatives"
+	antagpanel_category = "Starfury Operative"
+	job_rank = ROLE_TRAITOR
+	antag_hud_type = ANTAG_HUD_OPS
+	antag_hud_name = "synd"
+	antag_moodlet = /datum/mood_event/badass_antag //focused
+	var/datum/team/starfuryop/sbccrew
+	can_hijack = HIJACK_HIJACKER
+
+/datum/antagonist/starfuryop/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	add_antag_hud(antag_hud_type, antag_hud_name, M)
+
+/datum/antagonist/starfuryop/remove_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
+	remove_antag_hud(antag_hud_type, M)
+
+/datum/antagonist/starfuryop/greet()
+	owner.current.playsound_local(get_turf(owner.current), 'sound/ambience/antag/ops.ogg',100,0)
+	owner.announce_objectives()
+
+/datum/antagonist/starfuryop/get_team()
+	return sbccrew
+
+/datum/antagonist/starfuryop/create_team(datum/team/starfuryop/new_team)
+	if(!new_team)
+		for(var/datum/antagonist/starfuryop/P in GLOB.antagonists)
+			if(!P.owner)
+				continue
+			if(P.sbccrew)
+				sbccrew = P.sbccrew
+				return
+		if(!new_team)
+			sbccrew = new /datum/team/starfuryop
+			sbccrew.forge_objectives()
+			return
```

Comment:
why are you checking this a second time?

Reply:
I actually have no idea, I'm not familiar with antag code, I'll check it all a bit later.